docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1717,8 +1749,890 @@ Here are the steps:
+   
+   NAME                              VERSION  IMAGE                    READY   PHASE      
+   --------------------------------  -------  -----------------------  ------  ---------  
+   'sample-domain1-admin-server'     '1'      'model-in-image:WLS-v2'  'true'  'Running'  
+   'sample-domain1-managed-server1'  '1'      'model-in-image:WLS-v2'  'true'  'Running'  
+   
+   @@ [2020-05-14T17:32:51][seconds=245] Info: '2' WebLogic pods currently match all criteria, expecting '3'.
+   @@ [2020-05-14T17:32:51][seconds=245] Info: Introspector and WebLogic pods with same namespace and domain-uid:
+   
+   NAME                              VERSION  IMAGE                    READY    PHASE      
+   --------------------------------  -------  -----------------------  -------  ---------  
+   'sample-domain1-admin-server'     '1'      'model-in-image:WLS-v2'  'true'   'Running'  
+   'sample-domain1-managed-server1'  '1'      'model-in-image:WLS-v2'  'true'   'Running'  
+   'sample-domain1-managed-server2'  '1'      'model-in-image:WLS-v2'  'false'  'Pending'  
+   
+   @@ [2020-05-14T17:32:52][seconds=246] Info: '2' WebLogic pods currently match all criteria, expecting '3'.
+   @@ [2020-05-14T17:32:52][seconds=246] Info: Introspector and WebLogic pods with same namespace and domain-uid:
+   
+   NAME                              VERSION  IMAGE                    READY    PHASE      
+   --------------------------------  -------  -----------------------  -------  ---------  
+   'sample-domain1-admin-server'     '1'      'model-in-image:WLS-v2'  'true'   'Running'  
+   'sample-domain1-managed-server1'  '1'      'model-in-image:WLS-v2'  'true'   'Running'  
+   'sample-domain1-managed-server2'  '1'      'model-in-image:WLS-v2'  'false'  'Running'  
+   
+   @@ [2020-05-14T17:33:25][seconds=279] Info: '3' WebLogic pods currently match all criteria, expecting '3'.
+   @@ [2020-05-14T17:33:25][seconds=279] Info: Introspector and WebLogic pods with same namespace and domain-uid:
+   
+   NAME                              VERSION  IMAGE                    READY   PHASE      
+   --------------------------------  -------  -----------------------  ------  ---------  
+   'sample-domain1-admin-server'     '1'      'model-in-image:WLS-v2'  'true'  'Running'  
+   'sample-domain1-managed-server1'  '1'      'model-in-image:WLS-v2'  'true'  'Running'  
+   'sample-domain1-managed-server2'  '1'      'model-in-image:WLS-v2'  'true'  'Running'  
+   
+   
+   @@ [2020-05-14T17:33:25][seconds=279] Info: Success!
+
+   ```
+     {{% /expand%}}
+
+1. After your domain roll is complete, you can call the sample web application to determine if the updated application was deployed. 
+
+   When the application is invoked, it should contain an output string like `Hello World! This is version 'v2' of the mii-sample JSP web-app.`.
+
+   Send a web application request to the Ingress controller:
```

Comment:
Ingress controller -> ingress controller (Capitalization change to match Kubernetes docs)

Reply:
done